server/cmd/mmctl/docs/mmctl_ldap_sync.rst

```diff
@@ -27,8 +27,8 @@ Options
 
 ::
 
-  -h, --help                      help for sync
-      --include-removed-members   Include members who left or were removed from a group-synced team/channel
```

Comment:
Should we keep the doc about --include-removed-members? It's deprecated but still works

Reply:
The docs are auto-generated based on code. I'm hesitant to tinker around too much with the questionable clarity we would get.